src/extensions/sonarcloud/overview.md

```diff
@@ -86,5 +86,7 @@ Learn More - [SonarQube Cloud | Azure integration][learnmore]
 [sc]: https://www.sonarsource.com/products/sonarcloud/?utm_medium=referral&utm_source=azure&utm_campaign=sc-product&utm_content=signup-sonarcloud-listing-x-x&utm_term=ww-psp-x
 [signup]: https://www.sonarsource.com/products/sonarcloud/signup/?utm_medium=referral&utm_source=azure&utm_campaign=sc-signup&utm_content=signup-sonarcloud-listing-x-x&utm_term=ww-psp-x
 [opensource]: https://sonarcloud.io/explore/projects?sort=-analysis_date?utm_medium=referral&utm_source=azure&utm_campaign=sc-signup&utm_content=signup-sonarcloud-listing-x-x&utm_term=ww-psp-x
-[getstarted]: https://www.azuredevopslabs.com/labs/vstsextend/sonarcloud/
+[getstarted]: https://github.com/microsoft/azuredevopslabs/blob/master/labs/vstsextend/sonarcloud/readme.md
```

Comment:
[Azure Labs will sunset in 2027](https://learn.microsoft.com/en-us/answers/questions/1818522/azure-lab-services-will-be-retired-on-june-28-2027) and the existing webpage is already down. The replacement points to the .md source file for that tutorial.